homeassistant/components/roborock/manifest.json

```diff
@@ -1,6 +1,9 @@
 {
   "domain": "roborock",
   "name": "Roborock",
-  "integration_type": "virtual",
-  "supported_by": "xiaomi_miio"
+  "codeowners": ["@humbertogontijo", "@Lash-L"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/roborock",
+  "iot_class": "cloud_polling",
+  "requirements": ["python-roborock==0.1.7"]
```

Comment:
Looking at this on https://pypi.org/project/python-roborock/ it seems like this project needs a little more work (e.g. has TODOs, no links to github, etc) so can you do some work to get this into a healthier state? I ended up doing some digging and found https://github.com/humbertogontijo/python-roborock which appears to be the source and i see you're doing cleanup work. (I was surprised this was running such an an older vesion with upates in the last 24 hours)

Reply:
Yeah good point - I'll take some time to work on the pypi page tomorrow.

As far as the version being delayed, offline support is currently being added to the package, so there's a lot of rapid development, I was going to wait until we had it fully tested and functionaly before it got bumped